### PR TITLE
APM > .NET > add new integrations

### PR DESCRIPTION
**Pull request by andrewlock**
### What does this PR do?
Adds new supported integrations and mentions Arm64 support

### Motivation
We want to document the latest features

### Preview

(These don't seem to show my changes for some reason? 🤔)

https://docs-staging.datadoghq.com/andrewlock/apm-dotnet/new-features/tracing/setup_overview/compatibility_requirements/dotnet-core/
https://docs-staging.datadoghq.com/andrewlock/apm-dotnet/new-features/tracing/setup_overview/compatibility_requirements/dotnet-framework/

### Additional Notes
---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by apigirl — APPROVED** [review]

File: content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ further_reading:
   - .NET Core 3.1
   - .NET Core 2.1
 
+- ARM64 processors are supported for .NET 5 on Linux only.
```
Comment (lucaspimentel) [inline content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md:28]: We should probably clarify that this line about ARM64 only applies to automatic instrumentation.

File: content/en/tracing/setup_overview/compatibility_requirements/dotnet-framework.md [hunk 2]
```diff
@@ -43,9 +43,11 @@ The .NET Tracer can instrument the following libraries automatically:
 | ADO.NET                         | built-in                       | `AdoNet`             |
 | HttpClient / HttpMessageHandler | built-in                       | `HttpMessageHandler` |
 | WebClient / WebRequest          | built-in                       | `WebRequest`         |
+| Microsoft Message Queuing (MSMQ)| `System.Messaging` 4.0+        | `Msmq`               |
```
Comment (lucaspimentel) [inline content/en/tracing/setup_overview/compatibility_requirements/dotnet-framework.md:46]: I think `System.Messaging.dll` is built into .NET Framework (see the few integrations above with `built-in`).

File: content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md (hunk 1 above)
Comment (lucaspimentel) [reply content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md:28]: We support the .NET runtimes listed above on the following architectures:
- win-x86
- win-x64
- linux-x64
- linuxmusl-x64 (is this the correct moniker?)
- linux-arm64 (only on .NET 5, added in v1.27.0)

Comment (andrewlock) [reply content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md]: Done in https://github.com/DataDog/documentation/pull/10719/commits/5aef47e11f7eaa740f46668bc89296b4e93e162e

File: content/en/tracing/setup_overview/compatibility_requirements/dotnet-framework.md (hunk 2 above)
Comment (andrewlock) [reply content/en/tracing/setup_overview/compatibility_requirements/dotnet-framework.md]: As these require customers to enable calltarget (which isn't documented) we will document these in a future release when CallTarget is enabled by default

File: content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md [hunk 3]
```diff
@@ -25,6 +25,13 @@ further_reading:
   - .NET Core 3.1
   - .NET Core 2.1
 
+- .NET Tracer supports the above runtimes on the following processor architectures
```
Comment (kevingosse) [inline content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md:28]: It's a bit weird to precise "the above runtimes", then for ARM "only .NET 5".

Maybe we can just say ".NET Tracer supports the following processor architectures" (with "the above runtimes" being implicit) ?

Comment (andrewlock) [reply content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md]: done in https://github.com/DataDog/documentation/pull/10719/commits/5d66faa5bd2d0020a575eb0674a1c7f954efedfb

**Review by kevingosse — APPROVED** [review]

**Review by apigirl — APPROVED** [review]

**Pull request merged**